### PR TITLE
New Template: ipinfusion_ocnos_show_lldp_table.textfsm

### PR DESCRIPTION
**Pull request by MheniMerz**
#### ISSUE TYPE
 - New Template Pull Request

#### COMPONENT
ipinfusion_ocnos_show_lldp_table.textfsm, ipinfusion_ocnos, show lldp]] table

#### SUMMARY

parse LLDP table for ipinfusion ocnos platform.  

#### NOTE
ipinfusion was not available in the [OS_CHOICES](https://github.com/networktocode/ntc-templates/blob/master/tests/test_index_order.py#L7) so i added it as it is supported by [netmiko](https://github.com/ktbyers/netmiko/blob/master/netmiko/ipinfusion/)

### CODE REVIEW

File: ntc_templates/templates/ipinfusion_ocnos_show_lldp_table.textfsm [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Value LOCAL_PORT (\S+)
```
Comment (jvanderaa) [inline ntc_templates/templates/ipinfusion_ocnos_show_lldp_table.textfsm:1]:
```suggestion
Value LOCAL_INTERFACE (\S+)
```

File: ntc_templates/templates/ipinfusion_ocnos_show_lldp_table.textfsm [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Value REMOTE_DEVICE (\S+)
```
Comment (jvanderaa) [inline ntc_templates/templates/ipinfusion_ocnos_show_lldp_table.textfsm:2]:
```suggestion
Value NEIGHBOR (\S+)
```

File: ntc_templates/templates/ipinfusion_ocnos_show_lldp_table.textfsm [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+Value REMOTE_PORT_DESC (\S+)
+
+Start
+  ^Capability\s+codes:\s+\(R\)\s+Router,\s+\(B\)\s+Bridge,\s+\(O\)\s+Other
+  ^LocalPort\s+RemoteDevice\s+RemotePortID\s+Capability\s+RemotePortDescr
+  ^[-+\s+]+ -> Row
+  ^. -> Error
+
+Row  
+  ^${LOCAL_PORT}\s+${REMOTE_DEVICE}\s+${REMOTE_PORT_ID}\s+${CAPABILITY}\s+${REMOTE_PORT_DESC} -> Record
```
Comment (jvanderaa) [inline ntc_templates/templates/ipinfusion_ocnos_show_lldp_table.textfsm:14]:
```suggestion
  ^${LOCAL_INTERFACE}\s+${NEIGHBOR}\s+${NEIGHBOR_INTERFACE}\s+${CAPABILITY}\s+${REMOTE_PORT_DESC} -> Record
```

File: ntc_templates/templates/ipinfusion_ocnos_show_lldp_table.textfsm [hunk 4]
```diff
@@ -0,0 +1,17 @@
+Value LOCAL_PORT (\S+)
+Value REMOTE_DEVICE (\S+)
+Value REMOTE_PORT_ID (\S+)
```
Comment (jvanderaa) [inline ntc_templates/templates/ipinfusion_ocnos_show_lldp_table.textfsm:3]:
```suggestion
Value NEIGHBOR_INTERFACE (\S+)
```

**Review by jvanderaa — CHANGES_REQUESTED** [review]
Recommended some changes to line up with the existing templates for LLDP Neighbors. This becomes very helpful when trying to work with structured data in a multi-vendor environment of having the same keys.

File: ntc_templates/templates/ipinfusion_ocnos_show_lldp_table.textfsm (hunk 1 above)
Comment (MheniMerz) [reply ntc_templates/templates/ipinfusion_ocnos_show_lldp_table.textfsm]: fixed in [ef74743](https://github.com/networktocode/ntc-templates/pull/923/commits/ef74743e064a2984fd28a620dddc342f8ab1534f)

File: ntc_templates/templates/ipinfusion_ocnos_show_lldp_table.textfsm (hunk 2 above)
Comment (MheniMerz) [reply ntc_templates/templates/ipinfusion_ocnos_show_lldp_table.textfsm]: fixed in [ef74743](https://github.com/networktocode/ntc-templates/pull/923/commits/ef74743e064a2984fd28a620dddc342f8ab1534f)

File: ntc_templates/templates/ipinfusion_ocnos_show_lldp_table.textfsm (hunk 3 above)
Comment (MheniMerz) [reply ntc_templates/templates/ipinfusion_ocnos_show_lldp_table.textfsm]: fixed in [ef74743](https://github.com/networktocode/ntc-templates/pull/923/commits/ef74743e064a2984fd28a620dddc342f8ab1534f)

File: ntc_templates/templates/ipinfusion_ocnos_show_lldp_table.textfsm (hunk 4 above)
Comment (MheniMerz) [reply ntc_templates/templates/ipinfusion_ocnos_show_lldp_table.textfsm]: fixed in [ef74743](https://github.com/networktocode/ntc-templates/pull/923/commits/ef74743e064a2984fd28a620dddc342f8ab1534f)

File: tests/test_index_order.py [hunk 5]
```diff
@@ -37,6 +37,8 @@
     "hp_comware",
     "hp_procurve",
     "huawei_vrp",
+    "ipinfusion",
```
Comment (jmcgill298) [inline tests/test_index_order.py:40]:
```suggestion
```
is there a reason for having `ipinfusion`? It looks like `ipinfusion_ocnos` should be sufficient

**Comment by jmcgill298** [comment]
replaced by https://github.com/networktocode/ntc-templates/pull/1251

**Pull request closed without merging**